tap/extensions/http/matcher.go

```diff
@@ -92,6 +92,6 @@ func splitIdent(ident string) []string {
 }
 
 func genKey(split []string) string {
-	key := fmt.Sprintf("%s:%s->%s:%s,%s", split[0], split[2], split[1], split[3], split[4])
+	key := fmt.Sprintf("%s:%s->%s:%s,%s%s", split[0], split[2], split[1], split[3], split[4], split[5])
```

Comment:
no "," needed?
```suggestion
	key := fmt.Sprintf("%s:%s->%s:%s,%s,%s", split[0], split[2], split[1], split[3], split[4], split[5])
```

Reply:
A delimiter is not really required since this is a unique identifier.